src/convertToObject.js

```diff
@@ -3,16 +3,32 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const classes = sourceString
+    .split(';')
+    .map(el => el.trim()
+      .split(':')
+      .map(i => i.trim()))
+    .filter(style => style.length > 1);
+
+  const result = classes
```

Comment:
`result` is not the best name for the variable
I would make it like 
```suggestion
  const convertedStyles = classes
```